### PR TITLE
Discovery server

### PR DESCRIPTION
**Pull request by hilary-luo**
Added documentation for the discovery server support and a little cleanup (readme tip, readme prettier from a previous change I did and using tabs and tables to make some of the config yaml pages a bit easier to go through for users)

### CODE REVIEW

File: docs_versioned_docs/version-ros2humble/ros/config/yaml/platform.mdx [hunk 1]
```diff
@@ -50,280 +53,86 @@ For systems without a BMS, the state of charge should be treated as a rough esti
...
-</details>
+<Tabs groupId="platform">
+  <TabItem value="A200" label="A200" default>
 
-<details><summary>J100</summary>
-<p>
+| Battery | Chemistry | Nominal Voltage (V) | Capacity (Ah) | Configurations |
+|:-:|:-:|:-:|:-:|:-:|
+| `ES20_12C` | Lead Acid | 12.0 | 20.0 | S2P1 |
+| `HE2613` | Lithium Ion | 25.9 | 12.8 | S1P3<br/>S1P4 |
```
Comment (rhysfaultless-cpr) [inline docs_versioned_docs/version-ros2humble/ros/config/yaml/platform.mdx:62]: @hilary-luo,  should _Lithium Ion_ change to one of these options?

- Lithium Iron Phosphate
- LiFePO₄
- LFP

File: docs_versioned_docs/version-ros2humble/ros/networking/ros2-networking.mdx [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+For more detailed information regarding Fast DDS or simple discovery, visit the
+[eProsima Fast DDS documentation](https://fast-dds.docs.eprosima.com/en/latest/fastdds/ros2/ros2.html).
+
+Additional details on how to select simple disocvery in the `robot.yaml` config file are available in the
+[Middleware section of Robot Configuration](../config/yaml/system.mdx#middleware).
+
+### Discovery Server
+
+A discovery server is a process running on a computer that serves to record and report ROS discovery information.
+A server can be though of as a look up table for ROS discovery information. When a participant (node) starts, it
```
Comment (rhysfaultless-cpr) [inline docs_versioned_docs/version-ros2humble/ros/networking/ros2-networking.mdx]: Typo: thought

File: docs_versioned_docs/version-ros2humble/ros/networking/ros2-networking.mdx [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+:::
+
+## Discovery Server Configurations
+
+There are multiple different ways to configure discovery server depending on which ROS 2 devices need to connect.
+This section will go over some basic configurations and is generalized for use as an introduction. It does not
+describe all possible configurations. The `ROS_DISCOVERY_SERVER` environment variable, shown in the various diagrams,
+indicates which discovery servers a given robot is connected to. Additionally, communication will be referred to
+in terms of seeing ROS 2 topics although this is representative of all nodes, topics, action servers, etc.
```
Comment (rhysfaultless-cpr) [inline docs_versioned_docs/version-ros2humble/ros/networking/ros2-networking.mdx]: Remove latin abbreviations, to improve language translations.

File: docs_versioned_docs/version-ros2humble/ros/networking/ros2-networking.mdx [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+  <figure>
+    <img
+      src={require("./img/ros2-fastdds-fully-connected-diagram.png").default}
+      width="500"
+    />
+  </figure>
+</center>
+
+This configuration is achieved by listing all of the computers' hostnames in the
+[`servers` section](../config/yaml/system.mdx#servers) on all of the robots. Including the remote PC allows
```
Comment (rhysfaultless-cpr) [inline docs_versioned_docs/version-ros2humble/ros/networking/ros2-networking.mdx]: Remove PC reference. "Computer" is preferred.
Refer to this comment for the background reasoning: 
https://github.com/clearpathrobotics/cpr-documentation/pull/81#discussion_r1116124370

**Comment by rhysfaultless-cpr** [comment]
There are a few references to `local PC` and `remove PC` that should be changed to Robot's Computer and Desktop Computer. This is related to this previous comment:
https://github.com/clearpathrobotics/cpr-documentation/pull/81#discussion_r1116124370

**Review by rhysfaultless-cpr — DISMISSED** [review]

File: docs_versioned_docs/version-ros2humble/ros/config/yaml/platform.mdx (hunk 1 above)
Comment (hilary-luo) [reply docs_versioned_docs/version-ros2humble/ros/config/yaml/platform.mdx:62]: I was just reorganizing how this information is displayed (formatting of tables and tabs) so I am not sure if that is desired or what the correct answer would be. That isn't the only battery that we report as Lithium Ion. I can ask around though.

Comment (tonybaltovski) [reply docs_versioned_docs/version-ros2humble/ros/config/yaml/platform.mdx:62]: @rhysfaultless-cpr which would you prefer?

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**